generators/client/templates/angular/_package.json

```diff
@@ -109,7 +110,7 @@
     <%_ } _%>
     "tslint": "5.5.0",
     "tslint-loader": "3.5.3",
-    "typescript": "2.5.2",
+    "typescript": "2.4.2",
```

Comment:
why is this downgraded?

Reply:
It is the `recommended` version

Reply:
Recommended by Angular? Doesnt it work with newer version?

Reply:
It is recommend by Angular. You can check it creating a project using angular cli and trying to upgrade typescript, you will see a warning.